test/studies/shootout/nbody/sidelnik/nbody_orig_1.chpl

```diff
@@ -107,6 +107,7 @@ proc main() {
                 -9.51592254519715870e-05 * days_per_year,
                 5.15138902046611451e-05 * solar_mass)
     ];
+  var bodies : [0..#NBODIES] borrowed Planet = ownBodies.borrow();
```

Comment:
Perhaps here `bodies` can be a `const` array of `owned Planet`.

Reply:
It could, but this test is a performance test that we graph, and I don't want to change its performance here